llvm/test/Transforms/Util/PredicateInfo/branch-on-same-cond.ll

```diff
@@ -9,24 +9,31 @@ define i32 @test(i32 %x) {
 ; CHECK-NEXT:    br label [[BB1:%.*]]
 ; CHECK:       bb1:
 ; CHECK-NEXT:    [[CMP:%.*]] = icmp eq i32 [[X:%.*]], 0
-; CHECK: RenamedOp: [[CMP]]
-; CHECK:         [[CMP_0:%.*]] = bitcast i1 [[CMP]] to i1
-; CHECK: RenamedOp: [[X]]
-; CHECK:         [[X_0:%.*]] = bitcast i32 [[X]] to i32
+; CHECK-NEXT:  ; Has predicate info
```

Comment:
I'd suggest dropping this from the PredicateInfo printing, it's rather redundant...